h/api/search.py

```diff
@@ -31,7 +34,41 @@ def _match_clause_for_uri(uristr):
     }
 
 
-def build_query(request_params):
+def scan(es_client, query, fields):
```

Comment:
One does have to know how ES works in order to use these. One has to first know about scanning and bulk operations, and about the query object format.